packages/ses/docs/lockdown.md

```diff
@@ -73,6 +87,14 @@ lockdown({ regExpTaming: 'safe' }); // Delete RegExp.prototype.compile
 lockdown({ regExpTaming: 'unsafe' }); // Preserve RegExp.prototype.compile
 ```
 
+If `lockdown` receives no election for `regExpTaming`, it will respect
```

Comment:
> receives no election

is a very strange turn of phrase. Is this an idiomatic usage of "election"? It might just be strange to my ears. I recommend getting a quick second opinion -- see if they understand what it means.

Reply:
It’s a _valid_ and pithy use of “election”, though I would not call it idiomatic. It’s also passive, so I’m inclined to try again regardless. Perhaps “If `lockdown` does not receive a `regExpTaming` option”, _ad nauseam_.

Reply:
> If lockdown does not receive a regExpTaming option

It is clearer